### PR TITLE
 Add support for variable groupings

### PR DESCRIPTION
**Pull request by leonklingele**
This adds support to parse the following construct:

	var (
		myVar1 = 1
		myVar2 = 2
		// ..
	)

### CODE REVIEW

**Comment by leonklingele** [comment]
@leighmcculloch sorry for basing this on #3, I don't want to open yet another PR for improving the layout of #3's tests.

File: testdata/7/code.go [hunk 1]
```diff
@@ -4,28 +4,28 @@ import (
...
+	// Those errors have a declared error type
+	declaredErr error = errors.New("declaredErr")
+	errDeclared error = errors.New("errDeclared")
+
+	// Those are actual errors which should be ignored
+	errUnexported = errors.New("errUnexported")
+	ErrExported   = errors.New("ErrExported")
+	// errCustomUnexported = customError{"errCustomUnexported"}
+	// ErrCustomExported   = customError{"ErrCustomExported"}
+)
```
Comment (leighmcculloch) [inline testdata/7/code.go]: 🎉 This is great. Can we make this `testdata/8` so that we have both top level and grouped vars being tested of this kind?

Comment (leonklingele) [reply testdata/7/code.go]: Shall I simply `cp -R testdata{7,8}` and make `testdata8` use grouped vars?

Comment (leighmcculloch) [reply testdata/7/code.go]: Yup I think that'd be perfect.

**Comment by leonklingele** [comment]
Updated.

**Comment by leonklingele** [comment]
Rebased to master, once again. Should be good to go now! :-)

**Comment by leighmcculloch** [comment]
Thanks @leonklingele !

**Comment by leonklingele** [comment]
Don't forget to push a new release and make the recent changes be available in `gometalinter` (that's how I discovered this project).

**Pull request merged**